src/Common/src/Internal/Runtime/ModuleHeaders.h

```diff
@@ -0,0 +1,23 @@
+// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
I'm conflicted about the location of this file. While it's nice to have both .cs and .h in a single places, we only put managed stuff into src/Common.

Everything native lives under src/Native, and we maintain that split even for common things like EEType flag definitions.